src/rpc/rawtransaction.cpp

```diff
@@ -367,7 +367,7 @@ UniValue getrawtransaction(const UniValue& params, bool fHelp)
 
 UniValue getrawcertificate(const UniValue& params, bool fHelp)
 {
-    if (fHelp || params.size() < 1 || params.size() > 2)
+    if (fHelp || params.size() < 1 || params.size() > 3)
         throw runtime_error(
             "getrawcertificate \"certid\" ( verbose )\n"
             "\nNOTE: By default this function only works sometimes. This is when the certificate is in the mempool\n"
```

Comment:
`\nNOTE: By default this function only works sometimes.`
With the changes you just performed, doesn't this function always work? You have option to look into the real blockchain now. If so, please rewrite help